compliance_checker/suite.py

```diff
@@ -22,8 +23,10 @@
 from netCDF4 import Dataset
 from owslib.sos import SensorObservationService
 from owslib.swe.sensor.sml import SensorML
-from packaging.version import parse
-from pkg_resources import working_set
+if sys.version_info >= (3, 10):
+    import importlib.metadata as impmd
+else:
+    import importlib_metadata as impmd
```

Comment:
It is a bit clearer this 
```suggestion
import importlib_metadata
```

Reply:
All the same as above -- I prefer to future-proof, and in the present not require a backport that isn't needed. But this isn't my baby.

NOTE: this one was a bit tricky -- 'cause `importlib.metadata` exists in 3.8+ (I think) but apparently the API changed more recently :-(

Reply:
Yeah. This was quite the churn and hard to follow. All these backpport libraries solved parts of the problem.